libc/string/strlwr.c

```diff
@@ -0,0 +1,11 @@
+#include <string.h>
+
+char* strlwr(char* str){
+	int len = strlen(str);
+	for(size_t index = 0; index < len; index++){
+		if(64 < str[index] && str[index] < 91){
```

Comment:
I prefer it this way although it is my personal opinion
```suggestion
		if(str[index] > 64 && str[index] < 91){
```